bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZJetsToNuNu/ZJetsToNuNu_Zpt-100to200_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model sm-ckm_no_b_mass
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > ve ve~ / h @0
+add process p p > ve ve~ j / h @1
+add process p p > ve ve~ j j / h @2
+add process p p > ve ve~ j j j / h @3
+add process p p > ve ve~ j j j j / h @4
```

Comment:
Is there a reason to only do "ve ve~" for ZJets while you do "vl vl~" for ZJets+b?

Reply:
Thanks, @sihyunjeon for noticing this, I have fixed this bug now